frontend/packages/dev-console/src/components/helm/HelmReleaseDetailsPage.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router';
+import {
+  navFactory,
+  ResourceSummary,
+  SectionHeading,
+  LoadingBox,
+  StatusBox,
+  FirehoseResult,
+} from '@console/internal/components/utils';
+import { SecretModel } from '@console/internal/models';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { DetailsPage } from '@console/internal/components/factory';
+import { K8sResourceKindReference } from '../../../../../public/module/k8s';
+
+const SecretReference: K8sResourceKindReference = 'Secret';
+const HelmReleaseReference = 'HelmRelease';
+
+export interface HelmReleaseDetailsPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+  secrets?: FirehoseResult;
+}
+
+export const HelmReleaseDetails: React.SFC<any> = ({ obj: resourceDetails }) => {
+  const helmRelease = resourceDetails;
+  helmRelease.metadata.name = resourceDetails.metadata.labels.name;
+
+  return (
+    <div className="co-m-pane__body">
+      <SectionHeading text="Helm Release Overview" />
+      <div className="row">
+        <div className="col-sm-6">
```

Comment:
nit: will be good to handle for `xs` so that anything hight will follow the same.

Reply:
I have followed what has been done in the Overview page of other resources. Did you see any unusual behaviour when the height changes ?